docs/docs/releases.md

```diff
@@ -163,13 +163,35 @@ echo "deb https://dl.cloudsmith.io/public/pomerium/pomerium/deb/debian buster ma
 brew tap pomerium/tap
 brew install pomerium-cli
 ```
+### Docker Image
+
+The CLI utilizes a [minimal](https://github.com/GoogleContainerTools/distroless) [docker container](https://www.docker.com/resources/what-container). You can find the Pomerium CLI image on [dockerhub](https://hub.docker.com/r/pomerium/cli). It can be pulled in several flavors and architectures.
+
+- `:vX.Y.Z`: which will pull the a [specific tagged release](https://github.com/pomerium/cli/tags).
```

Comment:
As above, there should be tags and releases before we make links to them.

Reply:
See [comment](https://github.com/pomerium/pomerium/pull/2790#discussion_r759496159).

We can hold this if needed but this is only going to be in the default site on the next release.  The CLI will have releases by then.